RPerformanceTracking/Private/_RPTTracker.m

```diff
@@ -102,6 +102,16 @@ - (void)end:(uint_fast64_t)trackingIdentifier
     }
 }
 
+- (void)end:(uint_fast64_t)trackingIdentifier statusCode:(NSInteger)statusCode
```

Comment:
Can we get the other version of this method (without statusCode) to just call this with a 0 statusCode param?